docs-kits/kits/Eco_Pass_KIT/page-software-operation-view.md

```diff
@@ -0,0 +1,455 @@
+---
+id: Operation View EcoPass Kit
+title: Operation View
+description: 'EcoPass Kit Operation View'
+---
+
+![EcoPass KIT Pictotogram](./resources/img/EcoPassKIT_pictogram.png)
+
+# EcoPass KIT
+
+Based on the information provided in this kit, it is possible to run and program against an infrastructure of Digital Product Pass the Catena-X-way. This infrastructure empowers Data Consumers to consume the network's data as agreed with each Data Provider and facilitated by an Operating Company. They run central and decentral services that allow them to discover each other, exchange information and contextualize it according to a standardized semantics.
+
+
+
+| Service Name          | Description                                                                                                                                                                                                             | Reference Implementation                                                                                                                                                                                                                                                                                     | [Standardized in](https://catena-x.net/de/standard-library) |
```

Comment:
One standard is linked, the others are not. We should commit to either one, linking or not linking. Since standards might change independently from the KIT, it might be worthwhile to not link the standards since links might end up in dead ends. However, a reader may not know where to find the corresponding standard and get lost.

Reply:
thanks for the review :)